src/protocols/secure_channel/CASESession.cpp

```diff
@@ -385,7 +418,15 @@ CHIP_ERROR CASESession::HandleSigmaR1(System::PacketBufferHandle && msg)
 
     uint32_t decodeTagIdSeq = 0;
 
-    ChipLogDetail(SecureChannel, "Received SigmaR1 msg");
+    ChipLogDetail(SecureChannel, "Received Sigma1 msg");
+
+    bool sessionResumptionRequested = false;
+    uint8_t resumptionID[kCASEResumptionIDSize];
+    MutableByteSpan resumptionIDSpan(resumptionID);
+    uint8_t initiatorResume1MIC[kTAGSize];
+    MutableByteSpan resume1MICSpan(initiatorResume1MIC);
+
+    SuccessOrExit(err = ParseSessionResumptionRequest(msg, sessionResumptionRequested, resumptionIDSpan, resume1MICSpan));
```

Comment:
Suggest rename to `IsResumptionRequestPresent` since its validity is actually parsed below.